views/github-configuration.hbs

```diff
@@ -28,7 +29,7 @@
           <span class="text-bold text-gray-dark"><a href="{{html_url}}">{{ account.login }}</a></span>
           {{#if admin}}
             <button class="install-link btn btn-info float-right"  data-installation-id="{{ id }}" type="submit">Install</button>
-            <a href="/github/subscriptions/{{ id }}" class="float-right my-1 px-1">Manage</a>
+            <button class="manage-link btn btn-info float-right"  data-installation-id="{{ id }}" type="submit">Manage</button>
```

Comment:
any reason we are converting this as button ? but I like we are making this as button :)

Reply:
I wanted to pass jiraHost to `/github/subscriptions/` call so that we can retrieve `githubHost` and change token using secrets for that githubHost in [ghaeMiddleware](https://github.com/integrations/ghae-jira/blob/release/jira-for-ghae/lib/frontend/ghae-client-middleware.js)

on click of this button I am adding function in `github-configuration.js` which will pass jiraHost